datumaro/plugins/yolo_format/extractor.py

```diff
@@ -654,3 +678,85 @@ def _load_one_annotation(
             ]
         ]
         return Skeleton(points, label=label_id)
+
+
+class YOLOv8ClassificationExtractor(YoloBaseExtractor):
+    def _get_subset_names(self):
+        return [
+            subset_name
+            for subset_name in os.listdir(self._path)
+            if osp.isdir(osp.join(self._path, subset_name))
+        ]
+
+    def _get_image_paths_for_subset_and_label(self, subset_name: str, label_name: str) -> list[str]:
+        category_folder = osp.join(self._path, subset_name, label_name)
+        image_list_path = osp.join(category_folder, YOLOv8ClassificationFormat.LABELS_FILE)
+        if osp.isfile(image_list_path):
+            with open(image_list_path, "r", encoding="utf-8") as f:
+                yield from (osp.join(subset_name, label_name, line.strip()) for line in f)
+
+        yield from (
+            osp.relpath(image_path, self._path)
+            for image_path in find_images(category_folder, recursive=True)
+        )
+
+    def _get_item_info_from_labels_file(self, subset_name: str) -> Optional[Dict]:
+        subset_path = osp.join(self._path, subset_name)
+        labels_file_path = osp.join(subset_path, YOLOv8ClassificationFormat.LABELS_FILE)
+        if osp.isfile(labels_file_path):
+            return parse_json_file(labels_file_path)
+
+    def _get_lazy_subset_items(self, subset_name: str):
+        subset_path = osp.join(self._path, subset_name)
+
+        if item_info := self._get_item_info_from_labels_file(subset_name):
+            return OrderedDict(
+                (id, osp.join(subset_name, item_info[id]["path"])) for id in item_info
+            )
+
+        return OrderedDict(
```

Comment:
Why do we need `OrderedDict` here? Since Python 3.7 regular `dict` is guaranteed to be ordered.

Reply:
Replaced it with regular dict everywhere in the file.